src/pages/Lesson7.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'react';
+import lesson7array from '../../public/lesson7array.png';
+import Slide1lesson7 from '../../public/Slide1lesson7.png';
+import Slide2lesson7 from '../../public/Slide2lesson7.png';
+import Slide3lesson7 from '../../public/Slide3lesson7.png';
+import Slide4lesson7 from '../../public/Slide4lesson7.png';
+import Slide5lesson7 from '../../public/Slide5lesson7.png';
+import Slide6lesson7 from '../../public/Slide6lesson7.png';
+import AppWrapper from '../components/AppWrapper';
+import NavButtons from '../components/NavButtons';
+import SlideShow from '../components/SlideShow';
+import { HeaderSections } from '../types/globalTypes';
+import '../styles/PageWrapper.scss';
+import '../styles/Lesson7.scss';
+
+const Lesson7: FC = () => {
+  const imageArray = [];
+
+  imageArray.push(<img src={Slide1lesson7} className="lesson7slides" />);
+  imageArray.push(<img src={Slide2lesson7} className="lesson7slides" />);
+  imageArray.push(<img src={Slide3lesson7} className="lesson7slides" />);
+  imageArray.push(<img src={Slide4lesson7} className="lesson7slides" />);
+  imageArray.push(<img src={Slide5lesson7} className="lesson7slides" />);
+  imageArray.push(<img src={Slide6lesson7} className="lesson7slides" />);
+
+  return (
+    <div>
+      <AppWrapper section={HeaderSections.LESSON_7}>
```

Comment:
You need to add lesson 7 to the HeaderSections enum or it won't show up.
<img width="1503" alt="image" src="https://github.com/uclaacm/parcel-pointers/assets/78634083/71f9ea4a-dabd-4e13-bab7-3c8a14e33784">